Makefile

```diff
@@ -26,3 +26,13 @@ check:
 	# Configure project's revive checks using .revive.toml
 	# See: https://github.com/mgechev/revive
 	@for d in $$(go list ./... | grep -v /vendor/); do revive -config .revive.toml $${d}; done
+
+# Do a quick compile only check of the tests and impliclity the
+# library code as well.
+test-binaries: cephfs.test errutil.test rados.test rbd.test
+test-bins: test-binaries
```

Comment:
Do we need both `test-binaries` and `test-bins`? It makes more sense to me to have

    TEST_BINARIES := cephfs.test errutil.test rados.test rbd.test

and refer to that at the `test-bins` target.

Reply:
I was going to name it test-bins but I thought that might not be as clear as fully spelled out 'binaries'. I wanted to use 'test-bins' because its shorter. :-)